action/cleanup.go

```diff
@@ -0,0 +1,81 @@
+package action
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"sync"
+
+	ctx "github.com/hortonworks/cloud-haunter/context"
+	"github.com/hortonworks/cloud-haunter/types"
+	log "github.com/sirupsen/logrus"
+)
+
+var defaultRetentionDays = 90
+
+type cleanupAction struct {
+	retentionDays int
+}
+
+func init() {
+	retentionEnv := os.Getenv("RETENTION_DAYS")
+	var retentionDays int
```

Comment:
```
retentionDays = defaultRetentionDays
if retentionEnv, ok := os.Lookupenv("RETENTION_DAYS"); ok {
		parsed, err := strconv.Atoi(retentionEnv)
		if err != nil {
			log.Fatalf("[CLEANUP] Failed to parse RETENTION_DAYS, err: %s", err)
		}
		retentionDays = parsed
}
```
Does the same in a nicer way.

Reply:
This would not cover the case that the env variable is empty. However I have modified it to a similar syntax, and added tests too

Reply:
```
strconv.Atoi: parsing "": invalid syntax
```